app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/model/TicketType.kt

```diff
@@ -20,7 +21,10 @@ data class TicketType(
         @SerializedName("ticket_type")
         var id: Int = 0,
         var price: Int = 0,
-        var description: String = ""
+        var description: String = "",
+        @Ignore
+        @SerializedName("payment")
+        var paymentInfo: Payment = Payment()
```

Comment:
This leads to a database change no? Missing the migrations + schema.json then...

Reply:
No, paymentInfo is not saved in the database. I don't think it's necessary since we download it whenever the user wants to buy a ticket and we don't need it later.
```@Ignore``` makes Room ignore it (doesn't change the schema and doesn't try to insert it)